golem-templates/templates/c/c-app-common/wit/common.wit

```diff
@@ -1 +1,7 @@
+// This file is used to make the 'wit' directory compatible with tools such as wit-deps.
```

Comment:
I think actually `wit-deps` does not require it :) just a `wit/deps.toml`. The original reason why I added this file was so that I can generate directly bindings for packages from this dir in the "common libs" rust crates. 

So let's replace it to `such as wit-bindgen` that's more true

Reply:
https://github.com/golemcloud/golem-cli/pull/153/commits/ca49a3b88ba9f72ca1e144338378cf76ad9a9427